gnome-initial-setup/pages/live-chooser/gis-live-chooser-page.c

```diff
@@ -0,0 +1,305 @@
+/* gis-live-chooser-page.h
+ *
+ * Copyright (C) 2016 Endless, Inc
```

Comment:
Has the company name officially changed from “Endless Mobile, Inc” as credited in all other files?